packages/usa-input-mask/src/styles/_usa-input-mask.scss

```diff
@@ -0,0 +1,26 @@
+@use "uswds-core" as *;
+
+.usa-input-mask {
+  position: relative;
+
+  &--content {
+    border: 1px solid transparent;
+    color: color("gray-50");
+    font-size: font-size($theme-form-font-family, $theme-body-font-size);
+    left: 2px;
```

Comment:
Digging into the visual alignment of the mask content and input content and found that this left addition causes some horizontal mismatch. Can we remove this?

Reply:
This is a quirky one since removing it results in its own horizontal mismatch. I think we should leave it as-is for now, but but alignment on a short list of fast-follow improvements